index.js

```diff
@@ -109,31 +98,25 @@ controller.hears(['hello'], ['message_received', 'direct_message', 'direct_menti
   bot.startConversation(message, askName)
 })
 
-let getUsers = function (bot) {
-  bot.api.users.list({}, (err, response) => {
-    if (err) return err
-    // console.log(response.members[0])
-    response.members.forEach((user) => {
+function getUsers (callback) {
+  web.users.list(token, (err, response) => {
+    if (!err) {
```

Comment:
Checking for the error first is the most universally accepted pattern when doing callback logic. Instead of this:

```js
if (!err) {
  // do stuff
  callback(null, result)
}
else {
  callback(err)
}
```

try this:

```js
if (err) {
  return callback(err)
}
// this part doesn't get called if an error occurred because of the return statement
// do stuff
callback(null, result)
```

From a readability/maintenance perspective most people expect the error checking logic first. Also by using the return keyword, the function immediately stops executing. This means we don't need the `else` portion of the if statement.